transformer/transformer.go

```diff
@@ -558,20 +563,20 @@ func transform(newArtifactsToProcess, allArtifacts []transformertypes.Artifact,
 	return pathMappings, newArtifactsCreated, nil
 }
 
-func runSingleTransform(artifactsToProcess, allArtifacts []transformertypes.Artifact, transformer Transformer, tconfig transformertypes.Transformer, env *environment.Environment, graph *graphtypes.Graph, iteration int) ([]transformertypes.PathMapping, []transformertypes.Artifact, error) {
+func runSingleTransform(artifactsToProcess, allArtifacts []transformertypes.Artifact, transformer Transformer, tconfig transformertypes.Transformer, env *environment.Environment, graph *graphtypes.Graph, iteration int) ([]transformertypes.PathMapping, []transformertypes.Artifact, int, error) {
```

Comment:
Would be good to use named returns in the function definition to make it readable on what the 3rd int value represents.

Reply:
Sure Ashok, thats vertex id, should I define the type vertexid int and change everywhere its being used?

Reply:
No. https://www.geeksforgeeks.org/named-return-parameters-in-golang/amp/